model.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs');
+
+class Model {
+    constructor(data) {
+        this.data = data;
+    }
```

Comment:
constructor pada model seharusnya berisi properti apa saja yang dimiliki sesuai data.json, misalnya this.id, this.task, this.status